coordinator/src/services/validation/bundle-validator.ts

```diff
@@ -170,6 +172,10 @@ export function verifyDispenseBundle(bundle: fhir.Bundle): Array<fhir.OperationO
     }
   })
 
+  if (medicationDispenses.some(medicationDispense => !getOrganisationPerformer(medicationDispense))) {
+    allErrors.push(errors.createMedicationDispenseMissingValueIssue("performer.actor"))
```

Comment:
fhir path performer.actor is not clear feedback to user that this is because there is a performer actor type of organization missing here. 

To confirm:
1) Mapping is correct
2) validation for this is necessary outside of fhir validator (if not, remove this check, if yes then make error clearer)
3) IG states we need Organization

Reply:
Have made this error feedback clearer for the moment, to discuss the solution for this in another ticket